Makefile

```
@@ -23,14 +36,44 @@ define compile_service
 	-o ${BUILD_DIR}/cocos-$(1) ./cmd/$(1)
 endef
 
-.PHONY: all $(SERVICES) install clean
+NVIDIA_ATTESTATION_HELPER_CARGO_ENV = $(if $(filter 1,$(NVAT_USE_SYSTEM_LIB)),NVAT_USE_SYSTEM_LIB=1,)
+NVIDIA_ATTESTATION_HELPER_RUSTFLAGS = $(strip $(RUSTFLAGS) $(if $(filter 1,$(NVAT_USE_SYSTEM_LIB)),,-C link-arg=-Wl,-rpath,$$ORIGIN/lib))
 
-all: $(SERVICES)
+.PHONY: all $(SERVICES) $(NVIDIA_ATTESTATION_HELPER) nvidia-attestation-helper-prereqs install clean
 
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)